src/copycat.test.ts

```diff
@@ -37,442 +37,443 @@ test('generated values', () => {
   expect(results).toMatchInlineSnapshot(`
     Object {
       "bool": Array [
-        true,
```

Comment:
The reason the other api methods are returning different results, is because I forgot to update these tests when upgrading to fictional `0.5.0` earlier 🤦 Which is reminds me that we need to get CI working for this repo.